layouts/docs/glossary.html

```diff
@@ -12,14 +12,14 @@ <h1>{{ .Title }}</h1>
 	<div class="tag-description invisible" id="placeholder">.</div>
 	{{ range (index site.Data "canonical-tags") }}
 	<div class="tag-description hide" id="{{ printf "tag-%s-description" .id  }}">
-		<i>{{ .description }}</i>
+		<i>{{ T .descriptionKey }}</i>
 	</div>
 	{{ end }}
-	{{ $sorted_tags := sort (index site.Data "canonical-tags") "name" }}
+	{{ $sorted_tags := sort (index site.Data "canonical-tags") "nameKey" }}
```

Comment:
Here, you can use [`sort`] to sort a map of ids to localized names. The Hugo function `sort` can sort by value.

However, for _this_ PR I suggest you leave the sorting as-is. A later PR can get that bit right.

Reply:
With the removal of the other keys, I had to change it to `id`. I think it will likely still sort in the same order, given that all the names and IDs were essentially the same, but can backtrack on this if needed.